mne/io/egi/tests/test_egi.py

```diff
@@ -198,6 +200,20 @@ def test_io_egi_mff():
     for ch in include:
         assert ch in raw.event_id
         assert raw.event_id[ch] == int(ch[-1])
+    # test converting stim triggers to annotations
+    if events_as_annotations:
+        # Grab the first annotation. Should be the first "DIN1" event.
+        assert len(raw.annotations)
+        onset, dur, desc, _ = raw.annotations[0].values()
+        assert np.isclose(onset, 2.438)
```

Comment:
Use this instead, the error is more informative
```suggestion
        assert_allclose(onset, 2.438)
```